src/controller/flags.gni

```diff
@@ -17,4 +17,7 @@ import("//build_overrides/chip.gni")
 declare_args() {
   # Build controller (set to false for a device)
   chip_support_commissioning_in_controller = true
+
+  # Build controller with webrtc python bindings
+  chip_support_webrtc_python_bindings = false
```

Comment:
Please make sure that this is set to true in CI, otherwise Python webrtc support will not be check by CI.

Reply:
You should include that now. Otherwise, you will have to state in the `Testing` section in PR description that this PR was tested manually. However, current statement is incorrect - running `scripts/build_python.sh` will not compile your changes...

Every PR needs to be tested, and if it is possible, we require automatic testing by CI. Adding, `-webrtc` option to the `build_examples.py` is simple enough that you should include it in that PR.

Also, please do not resolve conversation if it was not in fact resolved... This makes tracking review process really hard...